### PR TITLE
chore: License Correction

### PR DESCRIPTION
**Pull request by Karotoka**
Correct form of copyright based on advice from external counsel.

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -175,18 +175,7 @@
...
-      boilerplate notice, with the fields enclosed by brackets "[]"
-      replaced with your own identifying information. (Don't include
-      the brackets!)  The text should be enclosed in the appropriate
-      comment syntax for the file format. We also recommend that a
-      file or class name and description of purpose be included on the
-      same "printed page" as the copyright notice for easier
-      identification within third-party archives.
-
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2018-2023 Mustafa Al-Bassam
```
Comment (rootulp) [inline LICENSE:178]: [silly question] does this mean we have to update `2023` every year from now?

**Review by rootulp — APPROVED** [review]

**Comment by musalbas** [comment]
This isn't how you use the LICENSE file correctly so external counsel is 100% wrong. You're not supposed to edit the LICENSE file; that appendix is instructions about how to add a notice. 

The notice, as explained in the appendix, should be added either to a separate NOTICE file (preferred), or at the top of copyrighted source files.

**Comment by musalbas** [comment]
Here's an example from an Apache projects: https://github.com/apache/pulsar/blob/master/NOTICE

We can see the appendix in the LICENSE file is unmodified: https://github.com/apache/pulsar/blob/master/LICENSE#L179

**Comment by musalbas** [comment]
More context here: https://infra.apache.org/licensing-howto.html
image

**Comment by Karotoka** [comment]
On Mus' instruction.

**Pull request closed without merging**